internal/services/databricks/databricks_workspace_resource.go

```diff
@@ -335,49 +335,126 @@ func resourceDatabricksWorkspace() *pluginsdk.Resource {
 				},
 			},
 
+			"enhanced_security_compliance": {
+				Type:     pluginsdk.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"automatic_cluster_update_enabled": {
+							Type:     pluginsdk.TypeBool,
+							Optional: true,
+							Default:  false,
+						},
+						"compliance_security_profile_enabled": {
+							Type:     pluginsdk.TypeBool,
+							Optional: true,
+							Default:  false,
+						},
+						"compliance_security_profile_standards": {
+							Type:         pluginsdk.TypeSet,
```

Comment:
Can you confirm whether the API preserves the ordering of these? If it does then this should be a List
```suggestion
							Type:         pluginsdk.TypeList,
```

Reply:
No, ordering is not preserved / guaranteed. The order does not have additional meaning.